kimchi/wasm/src/plonk_verifier_index.rs

```diff
@@ -20,6 +20,41 @@ use std::path::Path;
 use std::sync::Arc;
 use wasm_bindgen::prelude::*;
 
+pub struct SimpleFeatureFlags {
```

Comment:
I'm not totally in favor of adding a new layer of "complexity" with a new structure, which is only used to build `FeatureFlags`. I would suggest either we simplify `FeatureFlags` or add a new constructor or we remove this layer.

Reply:
Ok I can just move this logic into the function that computes the feature flags below. Custom constructor for FeatureFlags is probably the way to go. Note that pickles uses a representation equivalent to SimpleFeatureFlags